### ISSUE TITLE
feature: Periodic backups

### ISSUE THREAD

**Issue by garrett** [uxbox/uxbox#3910]
### Is your feature request related to a problem? Please describe.

I'd like to access previous versions of files, in case a lot of work was accidentally deleted.

### Describe the solution you'd like.

There's no way to make snapshots and go back to older versions of the file. There should be a way, possibly with daily (or even hourly) backups. Backups should only be snapshotted when things change.

I'd like this to be shown in the history. It should be possible to revert to a previous version, or at least download an older version.

**Comment by garrett** [comment]
I actually ran into this today and lost several weeks worth of files, because there's no history across sessions (https://github.com/penpot/penpot/issues/3909) and had no way to get back to yesterday's file (which is where most of the work was).

I did do some work today that mattered as well, but I (thankfully) have that still. But not _most_ of the things I've been working on for weeks.

(Previously: I made backups when I was done with a file... but this was some exploratory design work and I have to bounce between several different tasks.)
